index.html

```diff
@@ -3115,14 +3004,12 @@ <h3><dfn>Writing or pushing content</dfn></h3>
     </p>
     <p>
       Each {{NFCReader}} can filter the <a>NFC content</a> based on
-      data type, and the URL [= url/path =] of the
-      <a>browsing context</a> which has been saved to the <a>author type record</a>
-      of the <a>NFC content</a>.
+      data type, and a message identifier (URL) pattern.
     </p>
     <p>
       If you filter by URL [= url/path =], that means it will be matched against
```

Comment:
Consider reformulating this as well: perhaps remove adressing as "you", 
the sentence is not very clear to me.

Reply:
I mean the whole paragraph: what does it mean that all records are accepted